prov/efa/src/rxr/rxr_pkt_type_data.c

```diff
@@ -202,6 +202,7 @@ ssize_t rxr_pkt_send_data_mr_cache(struct rxr_ep *ep,
 	}
 	data_pkt->hdr.seg_size = (uint16_t)payload_size;
 	pkt_entry->pkt_size = payload_size + RXR_DATA_HDR_SIZE;
+	pkt_entry->x_entry = tx_entry;
```

Comment:
Is there a reason for this change? I already see this being set in the caller, `rxr_pkt_post_data()`.

Reply:
No need to have it. Removed in new revision.